### PR TITLE
Surge latest updating incorrectly

### PR DESCRIPTION
**Pull request by kopijunkie**
# Problem
Following 5.5.0 release, JS SDK version on Surge latest was not updated to 5.5.0 and still on 5.5.0-beta.2

# Solution
Use a more robust/strict check using a regex pattern so that if a git tag has any trailing labels after the version number (i.e. "-rc", "-beta", etc.), the deploy script should ignore it

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [ ] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [ ] Have new automated tests been implemented?
- [ ] Have new manual tests been written down?
- [ ] Have any new strings been translated?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-847-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-847-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://847-pr-onfido-sdk-ui-onfido.surge.sh

File: deploy.sh [hunk 1]
```diff
@@ -31,8 +31,10 @@ then
     #sort -V              ref: http://stackoverflow.com/a/14273595/689223
     #sort -t ...          ref: http://stackoverflow.com/a/4495368/689223
     #reverse with sed     ref: http://stackoverflow.com/a/744093/689223
-    #git tags | ignore release candidates | sort versions | reverse | pick first line
-    LATEST_TAG=`git tag | grep -v rc | sort -t. -k 1,1n -k 2,2n -k 3,3n -k 4,4n | sed '1!G;h;$!d' | sed -n 1p`
+    #git tag regex        explanation and tests: https://regex101.com/r/CjNA8f/2
```
Comment (stefaniacardenas) [inline deploy.sh]: Thanks for the comment! That _thing_ is so hard to read 😨

Comment (kopijunkie) [reply deploy.sh]: haha...feel free to use the link to the regex test for further explanation 😆

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-847-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-847-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://847-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-847-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-847-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://847-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**